iam/api-client/src/main/java/iam/snippets/RemoveMember.java

```diff
@@ -16,37 +16,58 @@
 package iam.snippets;
 
 // [START iam_modify_policy_remove_member]
-import com.google.api.services.cloudresourcemanager.v3.model.Binding;
-import com.google.api.services.cloudresourcemanager.v3.model.Policy;
+import com.google.iam.v1.Binding;
+import com.google.iam.v1.Policy;
+
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 
 public class RemoveMember {
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace the variables before running the sample.
+    // TODO: Replace with your policy, GetPolicy.getPolicy(projectId, serviceAccount).
+    Policy policy = Policy.newBuilder().build();
+    // TODO: Replace with your role.
+    String role = "roles/existing-role";
+    // TODO: Replace with your member.
+    String member = "user:member-to-add@example.com";
 
-  // Removes member from a role; removes binding if binding contains 0 members.
-  public static void removeMember(Policy policy) {
-    // policy = service.Projects.GetIAmPolicy(new GetIamPolicyRequest(), your-project-id).Execute();
+    removeMember(policy, role, member);
+  }
 
-    String role = "roles/existing-role";
-    String member = "user:member-to-remove@example.com";
+  // Removes member from a role; removes binding if binding contains no members.
+  public static Policy removeMember(Policy policy, String role, String member) {
```

Comment:
Adding comments at each step of this method would be helpful as it contains many removals and member additions.